Editing pass on starter template readme (#693)
risc0/risc0: templates/rust-starter/README.md

# RISC Zero Rust Starter Template

> Welcome to the RISC Zero Rust Starter Template! This template is intended to give you a starting point for building a project using the RISC Zero zkVM. Throughout the code are comments labelled `TODO` in places where we expect projects will need to modify the code.

Reviewer: Reviewing this makes me realize the original language is potentially confusion, is this clearer?
```suggestion
Welcome to the RISC Zero Rust Starter Template! This template is intended to give you a starting point for building a project using the RISC Zero zkVM. Throughout the code are comments labelled `TODO` in places where we expect projects will need to modify the code (including this document!).
```

Author: How about this? 
```suggestion
Welcome to the RISC Zero Rust Starter Template! This template is intended to give you a starting point for building a project using the RISC Zero zkVM. Throughout the template (including in this README), you'll find comments labelled `TODO` in places where you'll need to make changes.
```

## Questions, Feedback, and Collaborations
We'd love to hear from you on [Discord](https://discord.gg/risczero) or [Twitter](https://twitter.com/risczero).

> [Structure of a zkVM Application]: https://www.dev.risczero.com/zkvm/developer-guide/zkvm-app-structure

Reviewer: The `www.` is unnecessary and I think it's nicer without it.
```suggestion
[Structure of a zkVM Application]: https://dev.risczero.com/zkvm/developer-guide/zkvm-app-structure
```
